Assets/Scripts/PositionTracker.cs

```diff
@@ -54,7 +54,7 @@ private void OnTriggerEnter(Collider other)
 
             // Fetch collision Point
             RaycastHit hitInfo = new RaycastHit();
-            if(Physics.Raycast(transform.position, transform.forward, out hitInfo))
+            if(Physics.Raycast(transform.position, other.transform.position - transform.position, out hitInfo))
```

Comment:
While my code was not ideal I think to get a more accurate depiction of where the collision occured we should be using the rigidbody's velocity.

Currently the way you are shooting the ray cast means the racer will always go a lot more towards the center of the target than it actually should be so clipping the very edge of the target would always give us a better score than it should.

This isn't very important given that every racer would be granted that advantage but it is important to understand that and keep it in mind